plugins/woocommerce/client/blocks/assets/js/blocks/category-title/edit.tsx

```
@@ -0,0 +1,212 @@
+					onChange={ ( v ) => setTitle( v ) }
+					__experimentalVersion={ 2 }
+				/>
+			</ContainerElement>
+		) : (
+			<ContainerElement tagName={ TagName } { ...blockProps }>
+				<a
+					href={ link }
+					target={ linkTarget }
+					rel={ rel }
+					onClick={ ( event ) => event.preventDefault() }
+					dangerouslySetInnerHTML={ {
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)